drone/starlark/starlark.go

```diff
@@ -50,7 +50,7 @@ var Command = cli.Command{
 		cli.Uint64Flag{
 			Name:  "max-execution-steps",
 			Usage: "maximum number of execution steps",
-			Value: 50000,
+			Value: 100000,
```

Comment:
@eoinmcafee00 is this already a value that I can override somehow from the drone CLI command line?

If yes, then this PR is not needed.

If no, then should I code some way to allow this value to be overridden by environment variable `DRONE_STARLARK_STEP_LIMIT` like in PR https://github.com/drone/drone/pull/3134 ?

Reply:
This change isn't required. You should be able to do` --max-execution-steps 10000` when making the cli command.

Reply:
```
phil@phil-OptiPlex-3050:~/git/owncloud/core/apps-external/web$ drone starlark --max-execution-steps 123000
2021/09/03 16:53:00 starlark evaluation error:
Traceback (most recent call last):
  .drone.star:731:47: in main
  .drone.star:2620:24: in checkStarlark
Error: Starlark computation cancelled: too many steps
phil@phil-OptiPlex-3050:~/git/owncloud/core/apps-external/web$ drone starlark --max-execution-steps 124000
```

works - good. That repo generates more than 123,000 steps!!!